docs/tutorials/01-getting-started.md

```diff
@@ -0,0 +1,70 @@
+Want to install the color scheme and make tmux pastel? Great! Here's how.
+
+## Step 1: Clone this repository
+
+```bash
+mkdir -p ~/.config/tmux/plugins/catppuccin
+git clone https://github.com/catppuccin/tmux.git ~/.config/tmux/plugins/catppuccin/tmux
```

Comment:
This should align with the installation in the README where we say to checkout the latest tag (with release please comments making sure that it is up to date)